BlockEntity/BEFruitpress.cs

```diff
@@ -500,14 +512,8 @@ public void OnBlockInteractStop(float secondsUsed, IPlayer byPlayer)
 
             if (!CompressAnimActive) return;
 
-            if (secondsUsed >= 4.8f)

```

Comment:
I noticed that when I removed this and the earlier seconds used conditions the press started to act more predictably.  The server animation would flip out if I changed it to run at 0.5f instead of 1f before I removed these conditions, but removing them not only fixed that, it also fixed the issue where you could hold the crank down for 4.8 seconds and then let go of the mouse button and it would finish screwing by itself.  Or sometimes just clicking right click fast enough would get it to screw down all by itself from the top.  That felt unintended.